quinn-proto/src/endpoint.rs

```diff
@@ -861,6 +861,14 @@ pub struct TransportConfig {
     pub loss_reduction_factor: u16,
     /// Number of consecutive PTOs after which network is considered to be experiencing persistent congestion.
     pub persistent_congestion_threshold: u32,
+    /// Number of seconds of inactivity before sending a keep-alive packet
+    ///
+    /// Keep-alive packets prevent an inactive but otherwise healthy connection from timing out.
+    ///
+    /// 0 to disable, which is the default. Only one side of any given connection needs keep-alive
+    /// enabled for the connection to be preserved. Must be set lower than the idle_timeout of both
```

Comment:
Should we emit a warning if this happens?

Reply:
Via the logger? Good idea.